modules/mod_ginger_rdf/lib/js/embed.js

```diff
@@ -13,33 +14,56 @@
 
     function formatDate(date) {
         // TODO base this on user's locale
-        return date.toLocaleString();
+
+        var minutes = date.getMinutes();
+        var hour = date.getHours();
+        var year = date.getFullYear();
+        var day = date.getDate();
+        var month = date.getMonth() + 1;
+        var year = date.getFullYear();
+
+        return day+'/'+month+'/'+year+', '+hour+':'+minutes;
+        // return date.toLocaleString();
     }
 
-    function loadJsonLd(element, data) {
+    function loadJsonLd(element, data, url) {
+        var embedSize = element.getAttribute('embed-size') ? element.getAttribute('embed-size') : 'large';
+
+        // TODO make site title dynamic
         var json = JSON.parse(data),
             html =
-                '<section class="ginger-embed">' +
-                '    <style>' +
-                '        .ginger-embed { border: 3px solid greenyellow; }' +
-                '        .ginger-embed h1 { color: green; }' +
-                '    </style>' +
-                '    <h1>' + json['http://purl.org/dc/terms/title'] + '</h1>' +
-                '    <details> ' +
-                '        <summary>' + json['http://purl.org/dc/terms/abstract'] + '</summary>' +
-                '        <time class="published" datetime="' + json['http://purl.org/dc/terms/issued'] + '">' + formatDate(new Date(json['http://purl.org/dc/terms/issued'])) + '</time>' +
-                '        <p>' + json['http://purl.org/dc/terms/description'] + '</p>' +
-                '    </details>';
+                '<a href="'+url+'" class="ginger-embed embed-size-'+embedSize+'" target="_blank">';
+            if (json['http://purl.org/dc/terms/publisher']) {
+                html +=
 
+                '    <div class="ginger-embed__origin"> ' +
+                '       <h2 class="ginger-embed__origin-title">bron: '+json['http://purl.org/dc/terms/publisher']+'</h2>' +
+                '    </div> ' +
+                '    <div class="ginger-embed__origin-hover"> ' +
+                '       <h2 class="ginger-embed__origin-title">ga naar '+json['http://purl.org/dc/terms/publisher']+'</h2>' +
+                '    </div> ';
+            }
         if (json['http://xmlns.com/foaf/0.1/thumbnail']) {
-            html +=
-                '    <picture>' +
-                '        <img src="' + json['http://xmlns.com/foaf/0.1/thumbnail']['@id'] + '">' +
-                '    </picture>' +
-                '</section>';
-        }
+                html +=
+                    '    <picture>' +
+                    '        <img src="' + json['http://xmlns.com/foaf/0.1/thumbnail']['@id'] + '">' +
+                    '    </picture>';
+            }
+
+            html +='<link href="http://hart_amsterdammuseum.docker.dev/lib/css/build/ginger_embed.css" type="text/css" media="all" rel="stylesheet">' +
```

Comment:
Needed are three links: target page (and also widget content), site link and css link. 99% of the cases the base url will be the same.